mitodl/mit-learn
# Gives LR Drawer role "dialog" and a name.

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5418

### Description (What does it do?)
This PR:
1. Gives the RoutedDrawer and hence learning resource drawer role "dialog" with an accessible name.
2. Improves the loading skeleton for the drawer title a little bit.

### How can this be tested?
I suggest comparing screenreader behavior on RC (or prod) behavior with this branch. I used VoiceOver.

With a screenreader activated:
1. Open the Learning resource drawer on RC or prod.
    - When the drawer first opens, it will read something about the carousel slides.
2. Now on this branch:
   - screenreader should read something like "Dialog, [ResourceType] ResourceTitle]".
3. If you have https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd or equivalent installed, you could check that the drawer no longer triggers "All page content must be included in a landmark role".
